package.json

```diff
@@ -404,6 +404,7 @@
           "type": "object",
           "description": "Options to use with the debugger when launching for unit test debugging. Any launch.json option is valid here.",
           "default": {},
+          "scope": "machine",
```

Comment:
The machine scoping is because of `sourceFileMap`?

Reply:
That was my thinking, but I think I agree it's worth roaming the other parts. Will revert.